styles/styles.css

```
@@ -1502,6 +1540,16 @@ body {
   background-image: none;
   background-color: #ffffff;
   color: #000000;
+  margin-top: 73px;
+  margin-bottom: 143px;
+}
+
+.info-email-description-catalog{
```

```suggestion
.info-email-description-catalog{
```
```suggestion
.info-email-description-catalog {
```
следи за код стайлом